website/static/js/jquery.homeycombs.js

```diff
@@ -0,0 +1,145 @@
+(function($) {

```

Comment:
Just to make sure: you need to copy the whole file because it is not easily available on a CDN, right?